src/views/studio/studio-info.jsx

```diff
@@ -36,10 +36,7 @@ const StudioInfo = ({
 
 StudioInfo.propTypes = {
     isLoggedIn: PropTypes.bool,
-    studio: PropTypes.shape({
-        title: PropTypes.string,
-        description: PropTypes.description
-    }),
+    studio: PropTypes.shape({}), // TODO remove, just for <Debug />
```

Comment:
Looks like you meant to remove this

Reply:
Oh yeah in the future, its still useful to have that debug information visible at all times in the playground